src/components/SearchBox/SearchBox.js

```diff
@@ -220,6 +242,8 @@ class SearchBox extends React.Component {
       this.animationTimer();
       this.refs.headerSearchInputField.focus();
     }
+
+    return true;
```

Comment:
Y'know — I don't think this is necessary. If the handler doesn't return `false` the event should just keep propagating and bubbling up. If returning early you need to return `true` (or at least not return `false`), but otherwise I don't think it's necessary. I believe, anyway.

Reply:
Your are right! For the function to work, we actually don't need it. I added it there as Header component is using Eslint and it has consistent-return rule so it would complain if no `return` there.

https://blog.mojotech.com/consistent-returns/

Reply:
Ah gotcha. Thanks!